recipe/meta.yaml

```
@@ -1,16 +1,16 @@
-{% set version = "24.3.1" %}
+{% set version = "25.6.0" %}
 
 package:
   name: keyring
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/k/keyring/keyring-{{ version }}.tar.gz
-  sha256: c3327b6ffafc0e8befbdb597cacdb4928ffe5c1212f7645f186e6d9957a898db
+  sha256: 0b39998aa941431eb3d9b0d4b2460bc773b9df6fed7621c2dfb291a7e0187a66
 
 build:
   number: 0
-  skip: true  # [py<38]
+  skip: true  # [py<39]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

Should we add SETUPTOOLS_SCM_PRETEND_VERSION? Otherwise LGTM.

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
  script_env:
    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
```

---

Honestly, I feel like this is one of those things we should probably set for all builds in an activation script somewhere. It does nothing if you're not using setuptools_scm. Is there ever a time when we want to build with a version set but not have it embedded in the resulting wheel?

---

In this case, it's irrelevant because the version is correct in the logs:

```
  Source in $SRC_DIR has version 25.6.0, which satisfies requirement keyring==25.6.0 from file://$SRC_DIR

  Removed keyring==25.6.0 from file://$SRC_DIR from build tracker '/tmp/pip-build-tracker-q2fgt57s'
Created temporary directory: /tmp/pip-unpack-hmnz4u8s
Building wheels for collected packages: keyring
  Created temporary directory: /tmp/pip-wheel-4mo03v1g
  Destination directory: /tmp/pip-wheel-4mo03v1g
  Building wheel for keyring (pyproject.toml): started
  Building wheel for keyring (pyproject.toml): finished with status 'done'

  Created wheel for keyring: filename=keyring-25.6.0-py3-none-any.whl size=39084 sha256=8034083098e5bfc779a3ec59bfe334495dd90163b87c27e1cd35bda66ffabcc4
```

Yeah, as a broader solution for all packages with `setuptools_scm`, it makes sense to prevent implicitly setting v0.0.0.

---

Well, the logs are one thing. I was more concerned about a `_version.py` being created with something other than 25.6.0 in it. Luckily, `keyring` does not embed their version in their package.